docs/requirements.txt

```
@@ -2,3 +2,4 @@ numpy>=1.13.3
 https://download.pytorch.org/whl/cpu/torch-1.1.0-cp37-cp37m-linux_x86_64.whl
 sphinx-rtd-theme==0.4.0
 numpydoc==0.8.0
+Cython>=0.28.5
```

It is really strange how `setup.py` tries to build the wheel from source. I have seen this before but I have not figured out the root case of it.

I think another way to fix it is to install scikit-learn before `setup.py` gets to run:

```suggestion
scikit-learn>=0.19.1
```

---

That did it: https://readthedocs.org/projects/skorch/builds/12622129/